docs/How the library works/form-config.md

```diff
@@ -1,580 +0,0 @@
-    // widgets.
-    labels: {
-      chapter30: 'A readable description (Chapter 30)'
-    },
-
-    // This is a map of values to a component, some text, or some jsx. If your field
-    // is a radio widget, the content here will be shown underneath the radio button
-    // for that value when it is selected.
-    nestedContent: {
-      'value': <p>Some text</p>
-    },
-
-    // This is a string of class names that will be added to the widget for the current
-    // field. Similar to the default `classNames` property, but will put the class names
-    // on the input/select/etc element itself, rather than a surrounding `div`.
-    widgetClassNames: '',
-
-    // For Array fields, this is a component that is shown when the item in the array is
-    // being shown read-only on a normal form page (i.e. not on the review page).
-    viewField: RowViewComponent,
-
-    // If you want a field to only be shown when another field is true, set this option
-    // to the property name. It will wrap the fields in an ExpandingGroup component with
-    // the expandUnder field as the first question.
-    expandUnder: '',
-
-    // If you need to match on a specific value, you can use this option to specify a value
-    // that the expandUnder field's data should equal. 
-    expandUnderCondition: 'someValue',
-    // This can also be a function, which receives the expandUnder field's data as an argument
-    expandUnderCondition: (field) => field === 'someValue' || field === 'someOtherValue',
-
-    // If you're using the expandUnder option, you can set this option on the field specified
-    // by expandUnder and it will add classes to the div that wraps all of the fields when
-    // they are expanded. See cookbook for an example use case.
-    expandUnderClassNames: '',
-
-    // Set this if you want to hide this field on the review page.
-    hideOnReview: true || false,
-
-    // Set this if you want to hide this field on the review page when the field value is falsy
-    hideOnReviewIfFalse: true || false
-
-    // Function that conditionally hides fields in the form
-    // The index argument is provided if you use `ui:required` on data inside an array
-    hideIf: function (formData, index) {
-      return true || false;
-    }
-
-    // Function that conditionally replaces the current field's schema
-    // The index argument is provided if you use `ui:required` on data inside an array
-    updateSchema: function (formData, schema, uiSchema, index, pathToCurrentData) {
-      // This function should return an object with the properties you want to update
-      // It will not completely replace the existing schema, just update the individual
-      // properties
-      return {
-        type: 'string'
-      };
-    },
-
-    // Use this if you have an array field that should not be pulled out of the
-    // page its in and shown separately on the review page
-    keepInPageOnReview: true
-  }
-}
-```
-
-## Schemaform cookbook
```

Comment:
This went to https://github.com/usds/us-forms-system/pull/112/files#diff-cdbcd358defc8605fb20cb42cbb901f4R1.